docs/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting Started
+nav: 0.1
+---
+Welcome to the Jotai Getting Started tutorial!
+This tutorial will introduce you to Jotai and teach you how to start using it correctly.
+
+For this tutorial, we assume that you're using Jotai with React. The example is based on [a Create-React-App folder structure](https://create-react-app.dev/docs/folder-structure) so the first thing you have to do is:
+
+`npx create-react-app my-app`
+
+## Installation
+To install the latest stable version of Jotai, run the following command:
+
+`npm install jotai` or `yarn add jotai`
+
+## Provider
+
+Provider is to provide a state for component sub tree.
+This works just like the normal React Context.
+A good place to put this is in your root component:
+
+```jsx
+import { Provider, atom, useAtom } from 'jotai'
+
+const App = () => (
+  <Provider>
+    <Input />
+    <CharCount />
+    <Uppercase />
+  </Provider>
+)
+
+export default App
+```
+
+We'll implement the components mentioned above in the following section.
+
+## atom
+
+There's an exported function called `atom` that comes with jotai, which represents a piece of state. Atoms can be read from and written to from any component.
+
+To create an atom, all you need is to pass an initial value.
+
+```jsx
+const textAtom = atom('hello')
+const textLenAtom = atom((get) => get(textAtom).length)
+const uppercaseAtom = atom((get) => get(textAtom).toUpperCase())
```

Comment:
These look a repetitive example to what we have in Introduction.
So, I guess we are still not sure and open for what we should teach in getting started.

Reply:
I chose this example from the showcase because it's really straightforward and easy to understand how to implement Jotai, and for getting started we should aim for simplicity

Reply:
Let's wait for @sandren 's advice.